masonry_core/src/core/properties.rs

```diff
@@ -71,38 +91,55 @@ impl Properties {
 
 impl PropertiesRef<'_> {
     /// Returns `true` if the widget has a property of type `P`.
+    ///
+    /// Does not check default properties.
     pub fn contains<P: Property>(&self) -> bool {
         self.map.contains::<P>()
     }
 
-    /// Get value of property `P`, or `None` if the widget has no `P` property.
-    pub fn get<P: Property>(&self) -> Option<&P> {
-        self.map.get::<P>()
+    /// Get value of property `P`.
+    ///
+    /// If the widget has an entry for `P`, returns that entry.
+    /// If the default property set has an entry for `P`, returns that entry.
+    /// Otherwise returns [`Property::static_default()`].
+    pub fn get<P: Property>(&self) -> &P {
```

Comment:
This is more of a chained lookup than a `get`. Maybe rename.

Reply:
Any name suggestions?